tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleClientSuite.scala

```diff
@@ -640,7 +597,7 @@ class RapidsShuffleClientSuite extends RapidsShuffleTestHelper {
   test ("three requests spanning two bounce buffer lengths") {
     val bb = closeOnExcept(getBounceBuffer(1000)) { buff =>
       val rowCounts = Seq(40, 180, 100)
-      val ptrBuffs = rowCounts.zipWithIndex.map { case (c, ix) => makeRequest(ix + 1, c) }
+      val ptrBuffs = rowCounts.zipWithIndex.map { case (c, ix) => makeRequest(c) }
```

Comment:
```suggestion
      val ptrBuffs = rowCounts.map(makeRequest(_))
```

Reply:
This suggestion is still relevant, `zipWithIndex` is unnecessary since `ix` is unused.